pkgs/development/tools/pnpm/fetch-deps/default.nix

```diff
@@ -58,9 +61,11 @@
           pnpm config set side-effects-cache false
           # As we pin pnpm versions, we don't really care about updates
           pnpm config set update-notifier false
+          # Run any additional pnpm configuration commands that users provide.
+          ${prePnpmInstall}
           # pnpm is going to warn us about using --force
           # --force allows us to fetch all dependencies including ones that aren't meant for our host platform
-          pnpm install --frozen-lockfile --ignore-script --force
+          pnpm install --frozen-lockfile --ignore-script --force ${installFlags}
```

Comment:
I know this is technically unrelated to this PR, but there is a typo here, the parameter is called `--ignore-scripts`: https://pnpm.io/cli/install#--ignore-scripts